Client/src/components/layout/Form/Textarea/Textarea.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import styles from './Textarea.modules.css';
+import PropTypes from 'prop-types';
+
+function Textarea(props) {
+  return (
+    <textarea
```

Comment:
you can pass all props down below if their names are match: 
```
<textarea
  className={styles.textareaElement}
  {...props}
>
  {props.children}
</textarea>
```

Reply:
Oh, that's cool. Thanks